submissions/natashafir/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,325 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Booking tickets</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="wrapper">
+    <div class="hall">
+        <div class="screen"></div>
+        <div class="all-seats">
+            <div class="firsts-seats">
+
+                <input aria-label="row 1 seat 1" type="checkbox" id="seat-1-1" class="visually-hidden">
```

Comment:
add value attributes to your input tags
[checkbox value](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value)